### PR TITLE
Fixes #10

### LINKED ISSUE #10: Add images to steps
For every step it will be much easier to understand if images accompanied the text. 

- Add images for every step in the read-me file.
- All images should be place in an "images" directory in the repo
- All images should be aligned right of the preview

### PR DESCRIPTION
**Pull request by Vanessa-Kris**
Please if the images are too big let me know.

### CODE REVIEW

File: images/1.png
Comment (PromiseFru) [inline images/1.png:1]: Remove unused files

File: images/3.png
Comment (PromiseFru) [inline images/3.png:1]: Remove unused files

**Review by PromiseFru — CHANGES_REQUESTED** [review]
Hello @Vanessa-Kris

I've dropped some review comments. Please address the issues and I'll carry out another review before merging the pull request.

Thank you for contributing 🚀

**Comment by Vanessa-Kris** [comment]
Updated @PromiseFru

**Review by PromiseFru — APPROVED** [review]

**Pull request merged**